wiki/Tournaments/CWC/2018/en.md

```diff
@@ -0,0 +1,246 @@
+   -  This may change with the actual Group Stage setup.
+
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| -----: | -------- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals - Week 1 | O & AB, AC, AD |
+| Finals - Week 2 | AE, P, Q |
+
+**Winning conditions:**
+   - In Group Stage, you need to win 4 maps to win a match. (Best-of-7)
+   - In the Round of 16 and the Quarterfinals, you need to win 5 maps to win a match. (Best-of-9)
+   - In Semifinals, you need to win 6 maps to win a match. (Best-of-11)
+   - In the **Finals and Grand Finals**, you need to win 7 maps to win the match. (Best-of-13)
+
+### Match Instructions
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!catch, Team-Vs., Win Condition: 'ScoreV2'. Room name must be "CWC 2018: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+
+2. Players are free to select up to two warm-up maps. Using beatmaps with questionable content is prohibited. All maps must be osu!catch specific maps.
+3. Each captain can ban **one beatmap** to be selected from the pool. These beatmaps are not allowed to be picked by any team in the entire match.  
+4. Beatmap selection will alternate between each captain selecting a beatmap out of the map pool.
+5. Each captain must use "!roll" once in #multiplayer.
+   - The winner of the !roll starts picking the first beatmap of the match.
+   - The loser of the !roll starts banning.
+
+6. Captains may pick freely from any bracket.
+   - In case of a tie, the tiebreaker map must be played.
+
+7. Results of the Group Stage will be published via a Statistics sheet.
+
+### Mappool Instructions
+
+1. There will be 1 mappool for the Group Stage, 1 mappool for the Round of 16, 1 mappool for the Quarterfinals, 1 mappool for the Semifinals and 1 mappool for the Finals.
+   - Finals and Grand Finals use the same mappool.
+
+2. Each mappool consists of 4 brackets: NoMod, [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers) and [DoubleTime](/wiki/Game_Modifiers).
+3. Each mappool consists of 16 maps in total.
+4. Each mappool has one tiebreaker.
+5. The NoMod bracket will be played with no modes activated.
+6. The Hidden, HardRock and DoubleTime bracket will be played with the respective modes activated.  
+   - When playing a map from the HardRock or DoubleTime pool, Hidden can be used optionally by any player.
+
+7. The tiebreaker will be played under NoMod conditions.
+8. The size of the NoMod bracket will be 6 in all stages.
```

Comment:
these are worded kinda wierd
"There will be 6 maps in the NoMod bracket for all stages." is clearer